### DISCUSSION TITLE
Integration in XCSoar - Question about Licensing

### DISCUSSION THREAD

**Discussion by lordfolken** [openAIP/openaip-public#187] [General]
Hi, 

 I currently maintain https://github.com/xcsoar/xcsoar-data-content. This is a backend for hosting / referencing 3rd party resources for the XCSoar application. 
I maintain currently in there: 
 - Airspace
 - Waypoints
 - Maps
 - RASP
 - Flarmnet DBs

This repo was created as a reaction to the deprecation of WELT2000 a few years back. 

I ignored OpenAIP mostly due to the CC-by-NC license. 

While XCSoar is FOSS software and will stay that way, its possible that some people create commercial products. This has happened for example in [SteFly's Openvario](https://stefly.aero). 

So here my basic Question: What would happen if the data of openaip is included / made available on a commercially sold device such as the openvario. Would this be a violation? Would we potentially expose derivative Projects to litigation by including OpenAIP data in the repo?

Regards,
 - Folken

**Comment by reskume** [comment]
Hi Philipp,

we still have the same license but we don't have any problems with OpenAIP
data being used within commercial apps and hardware. In fact many
commercial apps use our data. The point is that only selling this data
exclusively eg as 'a monthly data" update is not permitted. The reason for
this is that we don't want that other people monetize directly on our
contributers work by simply downloading and selling openaip to their
customers.

So I see no reason no to use it. With our new API you can also provide
custom datasets to your users now. We would be happy to see xcsoar use
parts of the openaip dataset. If you need any more help with the
integration using the direct download via GCP bucket or using the API
please let me know. I'm here to help.

Cheers,

Stephan



Philipp Wollschlegel ***@***.***> schrieb am Mi., 1. Juni
2022, 15:53:

**Comment by lordfolken** [comment]
Thank you Stephan for the concise answer.
